### DISCUSSION TITLE
Polygon mask

### DISCUSSION THREAD

**Discussion by lower-greenHand** [matteobruni/tsparticles#5162] [Q&A]
Hello！
    The tsParticles version is on 2.12.0,I want to implement Polygon mask on the framework of Vue3. Discovery has been unsuccessful and there is no polygon masking. The particles are scattered. Has it been discontinued in this version?

**Comment by matteobruni** [comment]
https://github.com/tsparticles/tsparticles/issues/4845#issuecomment-1336842887
